tests/test_hep_bdFFT.py

```diff
@@ -108,7 +172,57 @@ def test_documents_from_FFT_ignores_context():
 
     result = hep.do(create_record(snippet))
 
-    assert '_fft' not in result
+    assert 'documents' not in result
+    assert 'figures' not in result
+
+
+def test_documents_from_FFT_does_not_require_s():
+    schema = load_schema('hep')
+    subschema = schema['properties']['documents']
+
+    snippet = (
+        '<datafield tag="FFT" ind1=" " ind2=" ">'
+        '  <subfield code="a">http://www.mdpi.com/2218-1997/3/1/24/pdf</subfield>'
+        '  <subfield code="d">Fulltext</subfield>'
+        '  <subfield code="t">INSPIRE-PUBLIC</subfield>'
+        '</datafield>'
+    )  # DESY harvest
+
+    expected = [
+        {
+            'description': 'Fulltext',
+            'url': 'http://www.mdpi.com/2218-1997/3/1/24/pdf'
+        }
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['documents'], subschema) is None
+    expected[0]['key'] = result['documents'][0]['key']
```

Comment:
you shouldn't cheat in tests like that. If you want reproducibility, you need to mock the random generator you are using.